### DISCUSSION TITLE
getaddrinfo ENOTFOUND host.docker.internal

### DISCUSSION THREAD

**Discussion by taholmes160** [Budibase/budibase#7651] [Help] [answered]
im using the solution hack of host.docker.internal b/c my lamp stack is also in a docker container -- however im still getting this error : getaddrinfo ENOTFOUND host.docker.internal

im using docker compose for both the lamp stack and budibase -- im running it all on Ubuntu Linux 22.04

Can someone please point me the right direction?

Thanks
TIM

**Comment by melohagan** [comment, marked as answer]
Hi Tim, 

This is because in Linux you need to use `172.17.0.1` instead of host.docker.internal

**Reply by AnujChhikara** [reply]
thank you so much @melohagan I was struggling with this from last one day, really appreciate. Thank you so much ❤️
